lib/sheet.js

```diff
@@ -43,14 +50,39 @@ const setCell = (target, prop, value) => {
   return true;
 };
 
-class Sheet {
+class Sheet extends EventEmitter {
   constructor() {
+    super();
     this.data = new Map();
     this.expressions = new Map();
     this.values = new Proxy(this, { get: getValue });
     this.context = metavm.createContext(this.values);
     this.cells = new Proxy(this, { get: getCell, set: setCell });
   }
+
+  emit(eventName, ...args) {
```

Comment:
We do not need to override EE interface for this task

Reply:
By default emit return boolean value, but for identifier hook we want return value based on our own logic

Reply:
I proposed another solution in [f83a646](https://github.com/metarhia/metacalc/pull/26/commits/f83a6461782df347888fa82fd68d83a858b879f5)

Reply:
Also what behaviour we expect if user create two subscriptions on `identifier` event?

```js
const sheet = new Sheet();

sheet.on('identifier', () => 1);
sheet.on('identifier', () => 2);

console.log(sheet.values["A0"]) // returned value from which `cb` should be  here?
```

Reply:
We need to process all event handlers because we do not know which of them will return value. Here is an example: 
```js
const sheet = new Sheet();

sheet.on('identifier', (sheet, name) => {
  if (name === 'A0') return 1;
});
sheet.on('identifier', (sheet, name) => {
  if (name === 'A1') return 2;
});

console.log(sheet.values['A0']);
```

Reply:
@justgowa we can iterate all handlers, call them and find first that will return not undefined.